lib/bourbon.rb

```diff
@@ -1,20 +1,6 @@
 require "bourbon/generator"
 
 module Bourbon
-  if defined?(Rails) && defined?(Rails::Engine)
-    class Engine < ::Rails::Engine
-      require "bourbon/engine"
-    end
-
-    module Rails
-      class Railtie < ::Rails::Railtie
-        rake_tasks do
-          load "tasks/install.rake"
-        end
-      end
-    end
-  else
-    bourbon_path = File.expand_path("../../core", __FILE__)
-    ENV["SASS_PATH"] = [ENV["SASS_PATH"], bourbon_path].compact.join(File::PATH_SEPARATOR)
-  end
+  bourbon_path = File.expand_path("../../core", __FILE__)
```

Comment:
It looks like this was in `module Bourbon` because of the Railtie. Now that it's not, I'd move it out of the module entirely, probably right after the `require`s.